roles/keepalived/templates/keepalived.conf.j2

```diff
@@ -11,12 +11,18 @@ global_defs {
 {% if keepalived_set_script_security_flag %}
     enable_script_security
 {% endif %}
+{% if keepalived_enable_mail_notification | default(false) %}
```

Comment:
We could add this variable to `roles/keepalived/defaults/main.yml`. If we want to use the variable in more than one location this approach might be less error-prone.